install/configuring_inventory_file.adoc

```diff
@@ -59,6 +59,15 @@ xref:../architecture/core_concepts/containers_and_images.adoc#architecture-image
 Version Tag Policy] section in the Architecture Guide for more information.
 ====
 
+[discrete]
+[[advanced-install-image-version-policy]]
+=== Image Version Policy
+
+Images require a version number policy in order to maintain updates. See
+the
+xref:../architecture/core_concepts/containers_and_images.adoc#architecture-images-tag-policy[Image
+Version Tag Policy] section in the Architecture Guide for more information.
+
```

Comment:
This change seems unrelated?

Reply:
@sdodson Thank you. Nice catch. Not sure how that got in there. This was added to the Advanced Install document in 2017, but changed to a note by @adellape when he reorganized that topic. Not sure how it got back in. Will remove!